pages/common/ts.md

```diff
@@ -0,0 +1,20 @@
+# ts
+
+> Add timestamps to input lines.
+> More information: <https://joeyh.name/code/moreutils/>.
+
+- Add current timestamp to beginning of each line:
+
+`{{some_command}} | ts`
+
+- Add timestamps with microsecond precision:
+
+`{{some_command}} | ts "%b %d %H:%M:%.S"`
```

Comment:
```suggestion
`{{some_command}} | ts "{{%b %d %H:%M:%.S}}"`
```
Any user-provided input  should go in token syntax

Reply:
OK, if that's the rule in tldr, I'm fine with the change. I still think this line would be more helpful without brackets, because this specific use case (showing microseconds) is probably quite common, and it would be easier to just copy the line if there were no brackets.

Reply:
Those brackets won't be rendered. This is what this line will look like:
![image](https://user-images.githubusercontent.com/50295997/130359830-ef90a64f-c1fe-41b8-a281-932a12563849.png)

Reply:
Oh, that looks fine indeed! I use the `tldr` command from Ubuntu packages (which is https://github.com/psibi/tldr-hs), and it shows the brackets. With https://github.com/tldr-pages/tldr-python-client this looks much better.